modules/apps/frontend-js/frontend-js-loader-modules-extender/src/main/java/com/liferay/frontend/js/loader/modules/extender/internal/npm/builtin/BaseBuiltInJSModuleServlet.java

```diff
@@ -159,24 +159,26 @@ private void _sendResource(
 		if (inputStream == null) {
 			String moduleName = resourceDescriptor.getPackagePath();
 
-			if (extension.equals("map")) {
-				JSModule jsModule = jsPackage.getJSModule(
-					moduleName.substring(0, moduleName.length() - 7));
+			if (moduleName != null) {
```

Comment:
@izaera is there any chance that moduleName is null at this point? I think it is as long as `ModuleNameUtil.getPackagePath()` can return null.

This will end up returning a 404 which seems ok to me

Still in this case it sounds like we shall log something at debug level at least right?

Reply:
> is there any chance that moduleName is null at this point? I think it is as long as ModuleNameUtil.getPackagePath() can return null.

Yes, as you already guessed (when the module name is invalid, for example).

> This will end up returning a 404 which seems ok to me

If the module name is invalid, it should, I agree.

> Still in this case it sounds like we shall log something at debug level at least right?

Not sure about that. It all depends on how do we log 404 errors in other cases...

However, a debug log just in case someone  want to diagnose anything wouldn't hurt, I guess...

Reply:
@AlmirFe please add a debug log in case we find a null moduleName so that we make our lives easier in case we have to diagnose these things. Thanks